services/weatherService.js

```diff
@@ -0,0 +1,31 @@
+export class WeatherService{
+	selectCity(){
+		return document.querySelector('.weather__select').value;
+	}
+	showCountry(country){
+	  document.querySelector('.weather__info-country').textContent = `Country: ${country}`;
+	}
+	showCity(city){
+	  document.querySelector('.weather__info-city').textContent = `City: ${city}`;
+	}
+	showTemperature(temperature){
+	  document.querySelector('.weather__info-temperature').textContent = `Temperature: ${temperature} °C`;
```

Comment:
you can create method `changeTextContent` for example, and use it all over this service,

```
changeTextContent(selector, text) {
  document.querySelector(selector).textContent = text;
}

showTemperature(temperature) {
  const selector = '.weather__info-temperature';
  const textContent = `Temperature: ${temperature} °C`;
  
  this.changeTextContent(selector, textContent)
}

```